chat-plugins/roomsettings.js

```diff
@@ -303,6 +304,8 @@ exports.commands = {
 	inviteonlyhelp: [
 		`/inviteonly [on|off] - Sets modjoin +. Users can't join unless invited with /invite. Requires: # & ~`,
 		`/ioo - Shortcut for /inviteonly on`,
+		`/ionext OR /inviteonlynext - Sets your next battle to be invite-only.`,
```

Comment:
The abbreviation is conventionally second.

Reply:
It's weird because technically ionext has been the actual command and inviteonlynext is a new expansion of that, but changed